spec/test_helper.rb

```diff
@@ -6,12 +6,14 @@
 RSpec.configure do |config|
   original_stderr = $stderr
   original_stdout = $stdout
-  config.before(:all) do
+
+  config.before(:all, silent_cli: true) do
```

Comment:
I'm not sure if what I said was right. you may still need `:all`. Did it work ok (output from cli silenced) when you removed `:all`?